processes/github_access.md

```diff
@@ -20,8 +20,13 @@ All projects at Defra must be created under one of the organisations within the
 - [Defra](https://github.com/DEFRA) for software
```

Comment:
```
- [Defra](https://github.com/DEFRA) for software
```

Tricky, as if the Defra org is "for software" then it looks like, technically, the other orgs serve no purpose.  Wonder if it's possible to narrow the scope of the Defra org a little, or even bit it at the bottom and say it's for "all other software development"?

Reply:
Amended to say digital service development as hopefully that clarifies it a bit